README.md

```diff
@@ -1,31 +1,33 @@
 [![Documentation](https://img.shields.io/badge/doc-reference-blue)](https://docs.restate.dev)
 [![Examples](https://img.shields.io/badge/view-examples-blue)](https://github.com/restatedev/examples)
 [![Discord](https://img.shields.io/discord/1128210118216007792?logo=discord)](https://discord.gg/skW3AZ6uGd)
-[![Twitter](https://img.shields.io/twitter/follow/restatedev.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=restatedev)
+[![Slack](https://img.shields.io/badge/Slack-4A154B?logo=slack&logoColor=fff)](https://join.slack.com/t/restatecommunity/shared_invite/zt-2v9gl005c-WBpr167o5XJZI1l7HWKImA)
+[![Twitter](https://img.shields.io/twitter/follow/restatedev.svg?style=social&label=Follow)](https://x.com/intent/follow?screen_name=restatedev)
 
 # Restate - Building resilient applications made easy!
 
 <p align="center">
   <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="https://restate.dev/poster_intro_dark.svg">
-    <source media="(prefers-color-scheme: light)" srcset="https://restate.dev/poster_intro_2.svg">
-    <img alt="Restate overview" src="https://restate.dev/poster_intro_2.svg" width="500">
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/restatedev/img/refs/heads/main/restate-overview-light.png">
+    <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/restatedev/img/refs/heads/main/restate-overview-dark.png">
+    <img alt="Restate overview" src="https://raw.githubusercontent.com/restatedev/img/refs/heads/main/restate-overview-dark.png" width="500">
   </picture>
 </p>
 
-Easily build workflows, event-driven applications, and distributed services in a fault-tolerant manner with durable async/await.
+[Restate](https://restate.dev) is the simplest way to build resilient applications.
 
-[Restate](https://restate.dev) is great at building:
+Restate provides a distributed durable version of your everyday building blocks, letting you build a wide range of use cases:
 
-* [Lambda Workflows as Code](https://restate.dev/blog/we-replaced-400-lines-of-stepfunctions-asl-with-40-lines-of-typescript-by-making-lambdas-suspendable/)
-* [Transactional RPC Handlers](https://github.com/restatedev/examples/tree/main/end-to-end-applications/typescript/food-ordering)
-* [Event Processing with Kafka](https://restate.dev/blog/restate--kafka-event-driven-apps-where-event-driven-is-an-implementation-detail/)
-* [Much more](https://github.com/restatedev/examples)
+* [Workflows-as-Code](https://docs.restate.dev/use-cases/workflows)
+* [Microservice orchestration](https://docs.restate.dev/use-cases/microservice-orchestration)
+* [Event Processing](https://docs.restate.dev/use-cases/event-processing)
+* [Async Tasks](https://docs.restate.dev/use-cases/async-tasks)
+* [Agents, Stateful Actors, state machines, and much more](https://github.com/restatedev/examples)
```

Comment:
Wondering whether agents shouldn't be the top most item if we want to ride the hype train.